src/main/kotlin/io/github/nomisrev/routes/profile.kt

```diff
@@ -0,0 +1,42 @@
+package io.github.nomisrev.routes
```

Comment:
I had to add this for detekt, I'd prefer not to change the filename. Perhaps we should disable this rule in some other issue.

```suggestion
@file:Suppress("MatchingDeclarationName")

package io.github.nomisrev.routes
```

Reply:
Agreed, having to force every routes implementation to add the `@file:Suppress` for detekt isn't deal, something for a future PR 😄 

Addressed in [76bea8b](https://github.com/nomisRev/ktor-arrow-example/pull/186/commits/76bea8b64edebda3dd37fdf1973bc33f0b12a642)